docs/pytest.md

```diff
@@ -0,0 +1,55 @@
+# Using bktec with pytest
+To integrate bktec with pytest, you need to [install and configure Buildkite Test Collector for pytest](https://buildkite.com/docs/test-engine/python-collectors#pytest-collector) first. Then set the `BUILDKITE_TEST_ENGINE_TEST_RUNNER` environment variable to `pytest`, and specify the `BUILDKITE_TEST_ENGINE_RESULT_PATH` to define where the JSON result should be stored. bktec will instruct Test Collector to output the JSON result to this path, which is necessary for bktec to read the test results for retries and verification purposes.
+
+```sh
+export BUILDKITE_TEST_ENGINE_TEST_RUNNER=pytest
+export BUILDKITE_TEST_ENGINE_RESULT_PATH=tmp/pytest-result.json
```

Comment:
I don't like this setting as it looks like an abstraction leak, but it isn't a blocker.